src/darwin/Framework/CHIP/ServerEndpoint/MTRServerCluster.h

```diff
@@ -0,0 +1,110 @@
+/**
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <Matter/MTRAccessGrant.h>
+#import <Matter/MTRDefines.h>
+#import <Matter/MTRDeviceType.h>
+#import <Matter/MTRServerAttribute.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A representation of a server cluster implemented by an MTRDeviceController.
+ */
+MTR_NEWLY_AVAILABLE
+@interface MTRServerCluster : NSObject <NSCopying>
```

Comment:
I think we should remove `NSCopying` here as well

Reply:
Will do.

Reply:
Done.